client-mvp-04/src/sass/variables/_colors.scss

```diff
@@ -30,5 +30,8 @@ $link-accent-color: $accent-royal-blue-color;
 $border-color: $dark-secondary-color;
 
 // PROGRESS BAR COLORS
-$progress-item-default-color: #D3D0CB;
+$progress-item-default-color: #d3d0cb;
 $progress-item-active-color: $accent-royal-blue-color;
+
+// INPUT COLORS
+$radio-button-color: #32373b;
```

Comment:
Could you revert all these changes besides yours for lines 36-37?
Are those `prettier` changes?
If so I need to take a look before our `prettier/eslint configs` because I don't see that we need to reformat them.

Reply:
Yes, of course. I will revert those changes. Does that mean I have to revert to the previous version of my commit of this file?
Yes. Prettier is formatted them.

Reply:
You can just copy this file https://github.com/hackforla/VRMS/blob/development/client-mvp-04/src/sass/variables/_colors.scss 
and add your 2 lines of code